internal/kafka/internal/handlers/validation.go

```diff
@@ -94,6 +94,19 @@ func ValidateCloudProvider(kafkaService *services.KafkaService, kafkaRequest *db
 	}
 }
 
+//ValidateAZOption returns a validator that returns error if incorrect multiAZ value is used
+func ValidateAZOption(kafkaRequest *dbapi.KafkaRequest) handlers.Validate {
+	return func() *errors.ServiceError {
+		if kafkaRequest.InstanceType == "standard" && !kafkaRequest.MultiAZ {
```

Comment:
We could use the defined constants in `"internal/kafka/internal/kafkas/types/types.go"` instead of hardcoded strings e.g  "standard", "eval"

Reply:
Thanks @machi1990 I'll use the constants

Reply:
perfect. I forgot to mention the added tests too

Reply:
Thanks I forgot to add them there